src/main/java/core/basesyntax/abstracts/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.abstracts;
+
+import core.basesyntax.colosuply.ColorSupplier;
+import core.basesyntax.interfaces.AreaDefinition;
+import core.basesyntax.interfaces.Drawable;
+
+public abstract class Figure implements Drawable, AreaDefinition {
+    private String color;
+
+    public String getColor() {
+        final ColorSupplier colorSupplier = new ColorSupplier();
+        return colorSupplier.getRandomColor();
```

Comment:
Фігура має свій колір
```suggestion
        return color;
```